CoreFramework/Reporter/HtmlReport.cs

```diff
@@ -0,0 +1,214 @@
+﻿using AventStack.ExtentReports;
+using AventStack.ExtentReports.MarkupUtils;
+using AventStack.ExtentReports.Reporter;
+using CoreFramework.APICore;
+using CoreFramework.Reporter.ExtentMarkup;
+using NUnit.Framework;
+
+
+namespace CoreFramework.Reporter;
+
+internal class HtmlReport
+{
+    //private static int testCaseIndex;
+    //private static string testCaseName;
+    private static ExtentReports _report;
+    private static ExtentTest _extentTestSuite;
+    private static ExtentTest _extentTestCase;
+    
+
+    // ------------------------------- CREATE EXTENTREPORT  -------------------------------
```

Comment:
If you want to separate the class structure, please use the #region instead
![image](https://user-images.githubusercontent.com/46469814/204434990-8205a422-9ac8-4d1b-be38-d64d642ce841.png)

Reply:
Check again for the other files